tests/test_agents/test_registration.py

```diff
@@ -54,7 +54,7 @@ def slow_down_tests() -> Generator:
     log_messages.collection_complete.value,
     log_messages.request_update.value,
     log_messages.response_update.value,
-    "Entered in the 'finished_registration' round for period 0",
+    "local height == remote height; continuing execution...",
```

Comment:
So we report this when `reset_tendermint_with_wait` is called and executed successfully. 
Is this guaranteed to only occur right before degenerate round, not ever before, on successful execution? And is this reliably picked up on, in that it does not suffer from the same hiatus as the other message does?

https://github.com/valory-xyz/open-autonomy/blob/8a86d041a885fd0cd7dc460cfc1aa8165d09763f/packages/valory/skills/abstract_round_abci/behaviour_utils.py#L1705-L1709

Reply:
This message occurs earlier than the previous one we had but still ensures that we succeeded in the e2e test. It just decreases the chances. We still enter the degenerate round but we may have picked up the log message before the buffer fills since it is logged earlier.